cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "projectId": "atpc1m",
```

Comment:
Is this correct? Is it from an account at cypress?

Reply:
Yes, it's from the project at Cypress Dashboard I have created for this project